crypto/cipher_extra/e_aes_cbc_hmac_sha1.c

```diff
@@ -0,0 +1,362 @@
+/*
+ * Copyright 2011-2016 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <openssl/opensslconf.h>
+
+#include <stdio.h>
+#include <string.h>
+
+
+#include <openssl/aes.h>
+#include <openssl/cipher.h>
+#include <openssl/err.h>
+#include <openssl/evp.h>
+#include <openssl/objects.h>
+#include <openssl/rand.h>
+#include <openssl/sha.h>
+#include "../fipsmodule/aes/internal.h"
+#include "internal.h"
+
+#if defined(AES_CBC_HMAC_SHA_STITCH)
+
+typedef struct {
+  AES_KEY ks;
+  // Used to compute(init, update and final) HMAC-SHA1.
```

Comment:
Did you check these field usages and HMAC impl? I shared these context in the meeting on 03-28.

Reply:
the question is why do we need three `SHA_CTX`s? And even though I might now the answer because you shared it with me in a private meeting, that doesn't mean that other people will know it when they look at the code :) that's why I said it'd be good to elaborate a bit more here.

Reply:
The current comment follows general code style of this code base(see ssl struct). Would you add the suggested change(if available) here based on our latest knowledge transfer meeting? I am not sure what is still needed here because these field usages are distributed in both ctrl and cipher functions.